EasyPost.Tests/EasyPost.Tests.csproj

```diff
@@ -12,13 +12,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="EasyVCR" Version="0.3.1"/>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0"/>
-    <PackageReference Include="coverlet.collector" Version="1.2.0"/>
-    <PackageReference Include="Newtonsoft.Json" Version="[13.0.1, 14.0.0)"/>
-    <PackageReference Include="RestSharp" Version="[107.3.0, 108.0.0)"/>
-    <PackageReference Include="MSTest.TestFramework" Version="2.2.8"/>
-    <PackageReference Include="MSTest.TestAdapter" Version="2.2.8"/>
```

Comment:
Please reset this file to origin/master. The PR never touches this file.

Reply:
Looks like Rider auto formatted it, resolved.